notebooks/validation_notebooks/slsim_dc2_diffsky_galaxy_distribution_comparision.ipynb

```diff
@@ -0,0 +1,758 @@
+{
```

Comment:
<p>I seem to see still a step-function like behavior in the SLSim distributions. Can we make the redshift interval smaller?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
decreased the redshift step to 0.01 which will solve this.